bin/build-theme.js

```diff
@@ -1,22 +0,0 @@
-const path = require('path');
```

Comment:
pls keep in mind that this file is used in the `build-theme` and `build-theme:watch` scripts. These scripts should be updated accordingly.